src/libs/ajax/file-browser-providers/GCSFileBrowserProvider.ts

```diff
@@ -1,7 +1,8 @@
-import { Ajax } from 'src/libs/ajax';
 import FileBrowserProvider from 'src/libs/ajax/file-browser-providers/FileBrowserProvider';
+import { GoogleStorage } from 'src/libs/ajax/GoogleStorage';
 import { GCSItem } from 'src/libs/ajax/GoogleStorage';
```

Comment:
```suggestion
import { GCSItem, GoogleStorage } from 'src/libs/ajax/GoogleStorage';
```
Multiple imports, consider importing once.

Reply:
fixed